README.md

```diff
@@ -590,7 +586,7 @@ import json
 
 # GLOBAL VARS.
 apikey = 'xxxxx'
-cromwell_token = 'xxxx'
+cromwell_token = None
```

Comment:
@dapineyro do we want to silently support this or should this be completely removed from the docs mentions?